docs/reference/early-access.md

```diff
@@ -0,0 +1,56 @@
+---
+id: early-access
+title: Early access
+sidebar_label: Early access
+description: "Use early access releases to quickly learn about upcoming features and components."
+---
+
+Use early access releases to quickly learn about upcoming features and components.
+
+## Types of early access releases
```

Comment:
How does a user know if a feature is alpha, beta, or GA?
Docs? Release notes? In-app "alpha" or "beta" tags?
I strongly recommend in-app alpha and beta tags, so users working on most recent monthly release know where there's a risk of something breaking

Reply:
Agree, I propose to use in-app tags in the handbook: https://confluence.camunda.com/display/HAN/Early+Access+Releases (The UI should reflect the alpha status)

We will introduce such a tag in Web Modeler SM